lib/core/logger.js

```diff
@@ -6,7 +6,7 @@ module.exports = function createLoggers(app) {
   const loggerConfig = app.config.logger;
   loggerConfig.type = app.type;
 
-  if (app.config.env === 'prod' && loggerConfig.level === 'DEBUG' && loggerConfig.allowDebugAtProd === true) {
```

Comment:
I think default to false here is to keep the compatibility

Reply:
It is false as defaults.

Reply:
`loggerConfig.allowDebugAtProd !== true`